core/trino-spi/src/main/java/io/trino/spi/connector/ColumnHandle.java

```diff
@@ -15,4 +15,7 @@
 
 public interface ColumnHandle
 {
+    int hashCode();
+
+    boolean equals(Object obj);
```

Comment:
`obj` -> `other`

Reply:
add javadoc documenting what it needs to satisfy

for example

- can a table have two different columns that are equal?
- can two different tables have columns that are equal?
  - this is actually pretty important question; it's natural that column handle is named-based, so it's "scoped" within table